week3/assign1/src/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+const Button =(props)=>{
+
+  
+
+  return(
+    <Btn type="button" onClick={props.onClick} disabled={props.state === null}>{props.content}</Btn>
```

Comment:
위 suggestion에서처럼 props를 구조분해할당으로 받아오면 이런식으로 바로 사용 가능해서 코드가 단순화 될 수 있답니당
```suggestion
    <Btn type="button" onClick={onClick} disabled={state === null}>{content}</Btn>
```